circle.yml

```diff
@@ -0,0 +1,13 @@
+machine:
+  node:
+    version: 0.10.34
+
+test:
+  override:
+    - npm run audit
+    - npm run deps
+    - npm outdated --depth 0
+    - npm run lint
+    - node_modules/.bin/babel-node node_modules/.bin/isparta cover --report text --report html --verbose --dir ${CIRCLE_ARTIFACTS}/coverage test/index.js
```

Comment:
I made a separate script for this since it has some CircleCI specifics and I reckon it is better to keep it isolated here.